dartagnan/src/test/resources/spirv/workgroupsize-unset.spv.dis

```diff
@@ -0,0 +1,29 @@
+; @ Output: forall (%size[0]==2 and %size[1]==3 and %size[2]==4)
+; SPIR-V
```

Comment:
Isn't this missing the configuration? or is the intended behavior that the config is programmatically set by line 21 in the spirv code?

Reply:
Based on the [document](https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/WorkgroupSize.html) I think `WorkgroupSize` decoration shall only change the value of the `%gl_WorkGroupSize` (initialized with either `OpSpecConstantComposite` or `OpConstantComposite`) to the value of `WorkgroupSize` instead of setting value to `WorkgroupSize`. This test is to show when config is not given, the value of `%gl_WorkGroupSize` will not be updated.

Reply:
I still don't get it. How does the builtin relates to the config? I would expect that we need to sync between the values this variable gets and the config. For example, the `MP.spv.dis` one has `@Config: 2, 1, 1` and 
```
          %5 = OpSpecConstant %uint 1
          %6 = OpSpecConstant %uint 1
          %7 = OpSpecConstant %uint 1
%gl_WorkGroupSize = OpSpecConstantComposite %v3uint %5 %6 %7
```

Reply:
So in this example, the value of `%gl_WorkGroupSize` is [1,1,1], and the `threadGrid` should be [1,1,1] or [2,1,1]?